### PR TITLE
8273243: Fix indentations in java.net.InetAddress methods

### PR DESCRIPTION
**Pull request by AlekseiEfimov**
Hi,

The fix changes indentations in the following `java.net.InetAddress` methods:
- getAddressesFromNameService
- getHostFromNameService

It helps to improve code readability. Can't say the same about `getHostFromNameService` diffs shown in this PR.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8273243](https://bugs.openjdk.java.net/browse/JDK-8273243): Fix indentations in java.net.InetAddress methods

### Reviewers
 * [Daniel Fuchs](https://openjdk.java.net/census#dfuchs) (@dfuch - **Reviewer**)
 * [Brian Burkhalter](https://openjdk.java.net/census#bpb) (@bplb - **Reviewer**) ⚠️ Review applies to 27c03f4d329ecd47d09d890bc9f3a0b8011fcd8f

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/5336/head:pull/5336` \
`$ git checkout pull/5336`

Update a local copy of the PR: \
`$ git checkout pull/5336` \
`$ git pull https://git.openjdk.java.net/jdk pull/5336/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 5336`

View PR using the GUI difftool: \
`$ git pr show -t 5336`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/5336.diff">https://git.openjdk.java.net/jdk/pull/5336.diff</a>

</details>

### CODE REVIEW

**Review by dfuch — APPROVED** [review]
LGTM

File: src/java.base/share/classes/java/net/InetAddress.java [hunk 1]
```diff
@@ -657,46 +657,46 @@ public String getCanonicalHostName() {
...
 
-                //XXX: if it looks a spoof just return the address?
-                if (!ok) {
-                    host = addr.getHostAddress();
-                    return host;
-                }
-            } catch (SecurityException e) {
-                host = addr.getHostAddress();
-            } catch (UnknownHostException e) {
+            //XXX: if it looks a spoof just return the address?
```
Comment (bplb) [inline src/java.base/share/classes/java/net/InetAddress.java:689]: looks _like_ a spoof

**Review by bplb — APPROVED** [review]

File: src/java.base/share/classes/java/net/InetAddress.java (hunk 1 above)
Comment (AlekseiEfimov) [reply src/java.base/share/classes/java/net/InetAddress.java]: Thanks Brian. Changed it as suggested :)

**Review by dfuch — APPROVED** [review]

**Pull request closed without merging**